vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -129,10 +143,16 @@
       <DependentUpon>ABI\NativeTracing.idl</DependentUpon>
       <ObjectFileName>$(IntDir)\ABI\</ObjectFileName>
     </ClCompile>
-    <ClCompile Include="ABI\ReactNativeHost.cpp">
+    <ClCompile Include="..\Microsoft.ReactNative\ReactNativeHost.cpp">
```

Comment:
These should be included as a Shared project.
What's the project type of `Microsoft.ReactNative`?

Reply:
I considered that (and asked a question about this here: [https://teams.microsoft.com/l/message/19:ea7692fed931434db524836878adf9df@thread.skype/1589320958751?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8034415f-bc0a-4b5d-9453-78ee27e16b22&parentMessageId=1589320958751&teamName=React%20Native%20%40Microsoft&channelName=React%20Native%20Windows&createdTime=1589320958751](https://teams.microsoft.com/l/message/19:ea7692fed931434db524836878adf9df@thread.skype/1589320958751?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8034415f-bc0a-4b5d-9453-78ee27e16b22&parentMessageId=1589320958751&teamName=React%20Native%20%40Microsoft&channelName=React%20Native%20Windows&createdTime=1589320958751))

In essence, I'm wondering if it's worth the effort to create a shared project when, in the long run, we'd like to unify the two projects into a single "core" project.

---
In reply to: [427614660](https://github.com/microsoft/react-native-windows/pull/4924#discussion_r427614660) [](ancestors = 427614660)

Reply:
I believe it does. We do already have a common shared library (ReactWindowsCore) and a shared items project that can be used for these purposes (Shared).

But, it's certainly effortful and out of the scope of this change.

I'll look into these refactorings next week.